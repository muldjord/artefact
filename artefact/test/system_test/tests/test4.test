INPUT="querymany.xml"
EXPECT="test4_expected.xml"
FILTER="cat"
TITLE="Query on multiple data entries"
