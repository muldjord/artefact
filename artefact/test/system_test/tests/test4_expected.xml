<?xml version='1.0' encoding='utf-8'?>
<results>
  <result id="0" errcode="0" errstring="" class="lm600p"
          deviceid="testid" devicetype="lm600p" timestamp="">
    <group name="lm600p">
      <group name="odxt">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
      <group name="osin">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
    </group>
  </result>
  <result id="0" errcode="0" errstring="" class="lm600p"
          deviceid="testid" devicetype="lm600p" timestamp="">
    <group name="lm600p">
      <group name="odxt">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
      <group name="osin">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
    </group>
  </result>
  <result id="1" errcode="0" errstring="" class="tonoref2_refractometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_refractometer">
      <group name="odxt">
        <value name="sph">-0.50</value>
        <value name="cyl">-3.00</value>
        <value name="axis">89</value>
      </group>
      <group name="osin">
        <value name="sph">+1.75</value>
        <value name="cyl">-2.00</value>
        <value name="axis">63</value>
      </group>
    </group>
  </result>
  <result id="1" errcode="0" errstring="" class="tonoref2_keratometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_keratometer">
      <group name="odxt">
        <value name="r1">8.22</value>
        <value name="r2">7.78</value>
        <value name="axis">99</value>
      </group>
      <group name="osin">
        <value name="r1">8.18</value>
        <value name="r2">7.84</value>
        <value name="axis">59</value>
      </group>
    </group>
  </result>
  <result id="1" errcode="0" errstring="" class="tonoref2_tonometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_tonometer">
      <group name="odxt">
        <value name="mmhg">17.3</value>
      </group>
      <group name="osin">
        <value name="mmhg">17.7</value>
      </group>
    </group>
  </result>
  <result id="2" errcode="0" errstring="" class="visutron900plus"
          deviceid="testid" devicetype="visutron900plus" timestamp="">
    <group name="visutron900plus">
      <group name="odxt">
        <value name="sph_far">-7.50</value>
        <value name="sph_near">-7.50</value>
        <value name="cyl">-0.25</value>
        <value name="axis">120</value>
      </group>
      <group name="osin">
        <value name="sph_far">-5.00</value>
        <value name="sph_near">-5.00</value>
        <value name="cyl">-0.50</value>
        <value name="axis">152</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="tonoref2_refractometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_refractometer">
      <group name="odxt">
        <value name="sph">-0.50</value>
        <value name="cyl">-3.00</value>
        <value name="axis">89</value>
      </group>
      <group name="osin">
        <value name="sph">+1.75</value>
        <value name="cyl">-2.00</value>
        <value name="axis">63</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="tonoref2_keratometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_keratometer">
      <group name="odxt">
        <value name="r1">8.22</value>
        <value name="r2">7.78</value>
        <value name="axis">99</value>
      </group>
      <group name="osin">
        <value name="r1">8.18</value>
        <value name="r2">7.84</value>
        <value name="axis">59</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="tonoref2_tonometer"
          deviceid="testid" devicetype="tonoref2" timestamp="">
    <group name="tonoref2_tonometer">
      <group name="odxt">
        <value name="mmhg">17.3</value>
      </group>
      <group name="osin">
        <value name="mmhg">17.7</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="lm600p"
          deviceid="testid" devicetype="lm600p" timestamp="">
    <group name="lm600p">
      <group name="odxt">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
      <group name="osin">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="lm600p"
          deviceid="testid" devicetype="lm600p" timestamp="">
    <group name="lm600p">
      <group name="odxt">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
      <group name="osin">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
    </group>
  </result>
  <result id="3" errcode="0" errstring="" class="visutron900plus"
          deviceid="testid" devicetype="visutron900plus" timestamp="">
    <group name="visutron900plus">
      <group name="odxt">
        <value name="sph_far">-7.50</value>
        <value name="sph_near">-7.50</value>
        <value name="cyl">-0.25</value>
        <value name="axis">120</value>
      </group>
      <group name="osin">
        <value name="sph_far">-5.00</value>
        <value name="sph_near">-5.00</value>
        <value name="cyl">-0.50</value>
        <value name="axis">152</value>
      </group>
    </group>
  </result>
</results>
